### ISSUE TITLE
Update Persian translation

### ISSUE THREAD

**Issue by ro31337** [ro31337/libretaxi-www#24]
Therre is a file `index-fa.html` that seems like needs to be updated. Here is feedback from Persian native speaker:

===

...you have to get rid of some of the mistakes and make some modifications in that👇

بازمتن👈🏻 متن‌باز

بدون حساب اعتباری. پرداخت مستقیم با پول نقد. 👈🏻 پرداخت مستقیم با پول نقد و نه کارت اعتباری

هزینه‌هایشان را تنظیم کرده 👈🏻 کرایه‌ی پیشنهادیشان را اعلام کرده

محرمانگی 👈🏻 حریم شخصی
راب به 👈🏻 را به
خودتان 👈🏻 خودتان را
درامد 👈🏻 درآمد
انعطاف 👈🏻 انعطاف‌پذیری 

رهایی 👈🏻 کنارگذاشتن
مسافرت 👈🏻 تاکسی‌گرفتن و مسافرزدن
خدمت 👈🏻 خدمات
کاره 👈🏻 برنامه
مهربان با باتری 👈🏻 مصرف کم باتری

===

I don't know Persian, so can't understand and check these fixes. Somebody familiar with Persian language should check and apply fixes.

Thank you!

**Comment by danialbehzadi** [comment]
Yes, Some of them need to be corrected. I'll look at them tonight or tomorrow.

**Comment by danialbehzadi** [comment]
Done
